### PR TITLE
Small refactoring

### PR DESCRIPTION
**Pull request by deztructor**
Refactoring to improve readability and get rid of clang warnings.

(Flashed and verified that it it is working)

### CODE REVIEW

File: src/displayapp/screens/SystemInfo.cpp [hunk 1]
```diff
@@ -12,15 +12,16 @@
 #include "drivers/Watchdog.h"
 
 using namespace Pinetime::Applications::Screens;
+using DeviceTypes = Pinetime::Controllers::MotionController::DeviceTypes;
```
Comment (jonvmey) [inline src/displayapp/screens/SystemInfo.cpp:15]: If you're doing `using Foo = Some::Namespaces::Foo;` you can just do `using Some::Namespaces::Foo;` instead.

Comment (deztructor) [reply src/displayapp/screens/SystemInfo.cpp:15]: I can't because `Pinetime::Controllers::MotionController` is not a namespace or unscoped enum.

**Comment by Riksu9000** [comment]
Maybe instead of adding `WorkLoopCycle()`, the code that gets run once should be moved to `Process()` and a while loop should be added around `app->Work()` instead similar to how it is done in DisplayApp.

**Comment by deztructor** [comment]
> Maybe instead of adding `WorkLoopCycle()`, the code that gets run once should be moved to `Process()` and a while loop should be added around `app->Work()` instead similar to how it is done in DisplayApp.

It could be  done as a next step: the goal of this refactoring commit is to extract the code to decrease nesting, so it is easier to modify it in future. I wanted to make an obvious and easy to review change. Because as you see, this is the hot spot: conflicts appear while the review is pending, I already rebased it with conflicts when I did refactoring (for another feature I'd like to add), now there is a conflict again.

**Comment by Riksu9000** [comment]
This PR will be hard to merge, as it contains different opinionated changes and is outdated. The SystemTask change should be done by itself and in a single step, if it's worth doing. Will you update this PR accordingly, or should this be closed in favor of newer PRs @deztructor ?

**Comment by deztructor** [comment]
> This PR will be hard to merge, as it contains different opinionated changes and is outdated. The SystemTask change should be done by itself and in a single step, if it's worth doing. Will you update this PR accordingly, or should this be closed in favor of newer PRs @deztructor ?

Yeah, things moved on, so it would be better to close it. And things could be done in a new PR.

**Pull request closed without merging**